packages/pharos/assets/icons/filetype-zip.svg

```diff
@@ -0,0 +1 @@
+<path fill-rule="evenodd" clip-rule="evenodd" d="M0 0H11.4142L16 4.58579V16H0V0ZM2 2V14H14V5.41421L10.5858 2H2ZM9.66663 6V10.4444H10.8113V8.91076H11.1414C11.4996 8.91076 11.7845 8.89141 11.989 8.84895L11.9924 8.84818C12.1562 8.81076 12.3131 8.73539 12.4621 8.62712C12.6225 8.50996 12.7509 8.35189 12.8491 8.15869C12.953 7.95429 13 7.71189 13 7.44013C13 7.08585 12.9175 6.77853 12.7375 6.53427C12.5652 6.29857 12.346 6.13986 12.0822 6.06734C11.9102 6.01913 11.5751 6 11.1044 6H9.66663ZM11.8314 7.45122C11.8314 7.53646 11.8108 7.60341 11.7737 7.65945L11.7723 7.66163C11.7361 7.7179 11.6854 7.76109 11.613 7.79067C11.5843 7.8024 11.5264 7.81637 11.4281 7.82649C11.3328 7.8363 11.2104 7.84153 11.0594 7.84153H10.8113V7.06646H11.0091C11.2941 7.06646 11.4651 7.07625 11.5393 7.09092L11.5407 7.09118C11.6303 7.10813 11.6988 7.14843 11.7537 7.21209C11.8024 7.2685 11.8314 7.34312 11.8314 7.45122ZM7.44446 6H8.55557V10.4444H7.44446V6ZM6.33333 6H3V7.11111H4.65051L3 9.33333V10.4444H6.33333V9.33333H4.58792L6.33333 7.11111V6Z" fill="black"/>
```

Comment:
We also need to remove the `fill="black"` in the svg so that the icon can have other colors